docs/resources/ilm_policy.md

```diff
@@ -46,6 +46,7 @@ Optional:
 
 - `expiration` (String)
 - `filter` (String)
+- `noncurrentversionexpiration` (String)
```

Comment:
I think this is pulled from the issue, but it might be more clear for this to be `noncurrent_version_expiration_days`? It looks like the library also supports a count of versions in `NewerNoncurrentVersions`

Looking at the [source struct](https://github.com/minio/minio-go/blob/4c868ef0690c1201c2174f540c9e1c4a55fde673/pkg/lifecycle/lifecycle.go#L50) it also looks like this is just an int, but it seems like the CLI takes different options

Reply:
Thanks for the review, yes it was pulled from the issue.
I can modify that to days, as it only support a number of days actually.

For the NewerNoncurrentVersions I see that , but actually we don't need it on our organization so i didn't implement it.

Maybe we can do it on another PR?

Reply:
Yeah the newer versions doesn’t need to go in this PR, I think days alone is just fine. Thanks!